bonfire/config.py

```diff
@@ -54,17 +54,30 @@
 # set to true when bonfire is running via automation using a bot acct (not an end user)
 BONFIRE_BOT = os.getenv("BONFIRE_BOT")
 
-AUTO_ADDED_FRONTEND_DEPENDENCIES = (
+DEFAULT_FRONTEND_DEPENDENCIES = (
     "chrome-service",
     "frontend-configs",
     "rbac",
     "rbac-frontend",
     "host-inventory",
     "host-inventory-frontend",
-    "unleash-proxy"
+    "unleash-proxy",
 )
 
 
+def _parse_frontend_dependencies():
```

Comment:
suggest to rename as `_get_auto_added_frontend_dependencies`, or if you want to refactor where it is used in `process` too, maybe rename the variable as `FRONTEND_DEPENDENCIES` and the method as  `_get_frontend_dependencies`.

Reply:
Renamed it